Breaking change v0.8.5 -> v0.9.0

**Problem**
No longer compiles with [ncc](https://github.com/vercel/ncc). 

**How to reproduce**

1. Create file

`index.js`
```js
import { Table } from 'dynamodb-toolbox';

console.log(Table);
```

2. Then run `npx ncc build index.js -o lib`.

This works as expected with `v0.8.5`, but fails with `v0.9.0` with the error
```
Error: Module not found: Error: Default condition should be last one
Did you mean './dynamodb-toolbox'?
Requests that should resolve in the current directory need to start with './'.
Requests that start with a name are treated as module requests and resolve within module directories (node_modules).
```

**What is expected**
Would expect this to compile with ncc.

**Other notes**
I'm not sure what has changed exactly, but possibly has to do with the compiler options changing: https://github.com/jeremydaly/dynamodb-toolbox/compare/v0.8.5...v0.9#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0

Looking forward to your feedback! Cheers

--- a/package.json
+++ b/package.json
@@ -7,12 +7,12 @@
   "exports": {
     ".": {
       "import": {
-        "default": "./dist/esm/index.js",
-        "types": "./dist/esm/index.d.ts"
+        "types": "./dist/esm/index.d.ts",
+        "default": "./dist/esm/index.js"
       },
       "require": {
-        "default": "./dist/cjs/index.js",
-        "types": "./dist/cjs/index.d.ts"
+        "types": "./dist/cjs/index.d.ts",
+        "default": "./dist/cjs/index.js"
       }
     }
   },

